src/utils/Validation.js

```
@@ -0,0 +1,73 @@
+    }
+
+
+    /**
+     * Validates the unit
+     * @param {string} unit - The unit to validate.
+     * @returns {boolean} Returns true if the unit is valid; otherwise, false.
+     */
+    static isUnit(unit) {
+        return isValidBase64(unit, 44);
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)